### PR TITLE
[upnpcontrol] Fix javadoc links

### PR DESCRIPTION
**Pull request by mherwege**
- Avoiding imports purely for javadoc links
- Fix links to class methods

### CODE REVIEW

**Review by lsiepel — APPROVED** [review]
LGTM

File: bundles/org.openhab.binding.upnpcontrol/src/main/java/org/openhab/binding/upnpcontrol/internal/handler/UpnpHandler.java [hunk 1]
```diff
@@ -228,7 +228,8 @@ protected void updateStatus(ThingStatus status) {
 
     /**
      * Method called when a the remote device represented by the thing for this handler is added to the jupnp
-     * {@link RegistryListener} or is updated. Configuration info can be retrieved from the {@link RemoteDevice}.
+     * {@link org.jupnp.registry.RegistryListener RegistryListener} or is updated. Configuration info can be retrieved
```
Comment (wborn) [inline bundles/org.openhab.binding.upnpcontrol/src/main/java/org/openhab/binding/upnpcontrol/internal/handler/UpnpHandler.java:231]:
```suggestion
     * {@link org.jupnp.registry.RegistryListener} or is updated. Configuration info can be retrieved
```

File: bundles/org.openhab.binding.upnpcontrol/src/main/java/org/openhab/binding/upnpcontrol/internal/handler/UpnpHandler.java [hunk 2]
```diff
@@ -415,10 +416,10 @@ public void onStatusChanged(boolean status) {
     }
 
     /**
-     * This method wraps {@link org.openhab.core.io.transport.upnp.UpnpIOService.invokeAction}. It schedules and
-     * submits the call and calls {@link onValueReceived} upon completion. All state updates or other actions depending
-     * on the results should be triggered from {@link onValueReceived} because the class fields with results will be
-     * filled asynchronously.
+     * This method wraps {@link org.openhab.core.io.transport.upnp.UpnpIOService.invokeAction invokeAction}. It
```
Comment (wborn) [inline bundles/org.openhab.binding.upnpcontrol/src/main/java/org/openhab/binding/upnpcontrol/internal/handler/UpnpHandler.java:419]:
```suggestion
     * This method wraps {@link org.openhab.core.io.transport.upnp.UpnpIOService.invokeAction}. It
```

File: bundles/org.openhab.binding.upnpcontrol/src/main/java/org/openhab/binding/upnpcontrol/internal/handler/UpnpRendererHandler.java [hunk 3]
```diff
@@ -483,7 +483,9 @@ protected void getMediaInfo() {
 
     /**
      * Retrieves the current volume known to the control point, gets updated by GENA events or after UPnP Rendering
-     * Control GetVolume call. This method is used to retrieve volume by {@link UpnpAudioSink.getVolume}.
+     * Control GetVolume call. This method is used to retrieve volume with the
+     * {@link org.openhab.binding.upnpcontrol.internal.audiosink.UpnpAudioSink#getVolume UpnpAudioSink.getVolume}
```
Comment (wborn) [inline bundles/org.openhab.binding.upnpcontrol/src/main/java/org/openhab/binding/upnpcontrol/internal/handler/UpnpRendererHandler.java:487]:
```suggestion
     * {@link org.openhab.binding.upnpcontrol.internal.audiosink.UpnpAudioSink#getVolume}
```

File: bundles/org.openhab.binding.upnpcontrol/src/main/java/org/openhab/binding/upnpcontrol/internal/handler/UpnpServerHandler.java [hunk 4]
```diff
@@ -543,7 +543,8 @@ private void handleCommandInRenderer(ChannelUID channelUID, Command command) {
 
     /**
      * Add a renderer to the renderer channel state option list.
-     * This method is called from the {@link UpnpControlHandlerFactory} class when creating a renderer handler.
+     * This method is called from the {@link org.openhab.binding.upnpcontrol.internal.UpnpControlHandlerFactory
+     * UpnpControlHandlerFactory} class when creating a renderer handler.
```
Comment (wborn) [inline bundles/org.openhab.binding.upnpcontrol/src/main/java/org/openhab/binding/upnpcontrol/internal/handler/UpnpServerHandler.java:547]:
```suggestion
     * This method is called from the {@link org.openhab.binding.upnpcontrol.internal.UpnpControlHandlerFactory}
     * class when creating a renderer handler.
```

**Review by wborn — CHANGES_REQUESTED** [review]
I think there are still some issues:

**Comment by wborn** [comment]
Or does it show the text after the space  as string?

**Comment by wborn** [comment]
I see it works like that so I think only one link needs to be updated. I'll update my comments. :slightly_smiling_face:

File: bundles/org.openhab.binding.upnpcontrol/src/main/java/org/openhab/binding/upnpcontrol/internal/handler/UpnpHandler.java (hunk 2 above)
Comment (mherwege) [reply bundles/org.openhab.binding.upnpcontrol/src/main/java/org/openhab/binding/upnpcontrol/internal/handler/UpnpHandler.java]: Thanks for noticing

**Review by wborn — APPROVED** [review]
Thanks!

**Pull request merged**